frameworks/cassandra/tests/test_overlay.py

```diff
@@ -3,6 +3,8 @@
 
 import shakedown
 
+# Do not use import *; it makes it harder to determine the origin of config
+# items
```

Comment:
Not really. I did a review and the sum result of that review was a requested change. This was a comment in that review.